lib/iris/cube.py

```diff
@@ -62,7 +62,8 @@ class CubeMetadata(collections.namedtuple('CubeMetadata',
                                            'var_name',
                                            'units',
                                            'attributes',
-                                           'cell_methods'])):
+                                           'cell_methods',
+                                           'cell_measures_and_dims'])):
```

Comment:
> This is a really, really fine point made by @pp-mo that I'd never considered ...

Agreed, and since he'd done an excellent job of making his point I didn't feel I had much to add. But since you've asked for an explicit opinion... well, it's implicit in my "Agreed", but to remove any remaining ambiguity ...

> ... 'metadata' ... should *not* include cell_measures

:+1:

> It probably makes perfectly good sense to simply remove cell measures from all interpolated outputs

Only if the interpolation relates to one or more dimensions spanned by the cell measure?